### PR TITLE
feat - Input Documentation

### PR DESCRIPTION
**Pull request by Ayesha-7**

### CODE REVIEW

File: Styles/Components/input.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+.input .input-error {
+  border-color: var(--red-3);
+  background: var(--red-6);
+}
+
+.input .input-success:focus,
+.input .input-error:focus {
+  outline: none;
```
Comment (MaruthiPothuganti) [inline Styles/Components/input.css]: Making outline none for an input field is considered as bad practice when Accessibility is considered.

File: Docs/input.html [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+                type="text"
+                placeholder="Valid text input"
+                class="input-success"
+              />
+            </div>
+
+            <div class="input">
+              <label for="text">Password<sup>*</sup></label>
+              <input
+                type="text"
```
Comment (MaruthiPothuganti) [inline Docs/input.html]: For password's, type can be changed to password

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggested some changes. The UI is cool and Eye Soothing. Everything else looks great.

File: Styles/Components/input.css (hunk 1 above)
Comment (MaruthiPothuganti) [reply Styles/Components/input.css]: [Check here](https://stackoverflow.com/questions/19455499/why-is-outlinenone-not-considered-good-practice)

Comment (Ayesha-7) [reply Styles/Components/input.css]: Thanks! I didn't know this, removed outline:none.

File: Docs/input.html (hunk 2 above)
Comment (Ayesha-7) [reply Docs/input.html]: thanks for pointing out, made the changes.

Comment (Ayesha-7) [reply Docs/input.html]: can you please approve the PR, if everything looks good.

**Review by sharathnair9999 — APPROVED** [review]

**Review by sharathnair9999 — APPROVED** [review]

**Pull request merged**